### PR TITLE
List repositories for unknown issues

### PR DESCRIPTION
**Pull request by dLobatog**
Many issues are listed as part of the "Installer" project when in
reality they are part of the puppet modules. Therefore on every release,
I see a lot of 'unknown commits' - most of them belong to this category.

### CODE REVIEW

File: lib/tool_belt/cherry_picker.rb [hunk 1]
```diff
@@ -105,11 +105,24 @@ def log_entry(pick)
       "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
     end
 
-    def find_repository(revision)
-      @release_environment.repo_names.find do |repo_name|
-        @release_environment.commit_in_repo?(repo_name, revision)
+    def find_repository(revision, issue)
+      #@release_environment.repo_names.find do |repo_name|
```
Comment (ehelms) [inline lib/tool_belt/cherry_picker.rb]: Guessing you meant to drop this comment out?

File: lib/tool_belt/cherry_picker.rb [hunk 2]
```diff
@@ -105,11 +105,24 @@ def log_entry(pick)
...
+      repo_name = case issue['project']['name'].downcase
```
Comment (ehelms) [inline lib/tool_belt/cherry_picker.rb]: Should there be a default case? What if the name is not one of those below?

**Comment by dLobatog** [comment]
@ehelms Updated - if the name isn't one of those it will print the name (if found on the config file) - or the project-category

**Comment by ehelms** [comment]
So I am seeing this in the output, which seems oddly formatted:

```
Cherry Picks for repository: {"id"=>19, "name"=>"Katello"} - {"id"=>105, "name"=>"Installer"}
```

**Comment by jturel** [comment]
@dLobatog any plans for this one?

**Comment by ehelms** [comment]
Needs a rebase, and has sat stale. Going to close for now, please re-open if you get time to get back to it.

**Pull request closed without merging**